### ISSUE TITLE
no stored values in terminal42/contao-leads when using DBAFS

### ISSUE THREAD

**Issue by Baloobear** [terminal42/contao-fineuploader#100]
Hello,

when I use the fine uploader plugin in conjunction with your leads plugin I face the issue that the UUIDs (and/or the filenames) are not stored in the leads list entry when I activate the _use DBAFS_ option.

When I keep this option disabled the filenames of the uploaded files will be displayed in the leads entry.

Since I'm not into the DBAFS thing (and I'm not yet sure if I need this or better have to use it) I'm not sure if this is a real bug or a common technical issue.

Environment:
Contao: 5.1.7
contao-leads: 3.0.2
contao-fineuploader: 3.4.3

Kind regards,
Dennis

**Comment by LBeckX** [comment]
I get the same error

Environment:
Contao 4.13
contao-leads: 3.0.2
contao-fineuploader: 3.4.3

```log
[2023-06-20T11:58:42.967142+02:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred while executing a query: SQLSTATE[22007]: Invalid datetime format: 1366 Incorrect string value: '\xEE\xBD\xD6\x96\x00\x02...' for column `contao413`.`tl_lead_data`.`value` at row 1" at .../vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 117 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 1366): An exception occurred while executing a query: SQLSTATE[22007]: Invalid datetime format: 1366 Incorrect string value: '\\xEE\\xBD\\xD6\\x96\\x00\\x02...' for column `contao413`.`tl_lead_data`.`value` at row 1 at .../vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:117)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1366): SQLSTATE[22007]: Invalid datetime format: 1366 Incorrect string value: '\\xEE\\xBD\\xD6\\x96\\x00\\x02...' for column `contao413`.`tl_lead_data`.`value` at row 1 at .../vendor/doctrine/dbal/src/Driver/PDO/Exception.php:28)\n[previous exception] [object] (PDOException(code: 22007): SQLSTATE[22007]: Invalid datetime format: 1366 Incorrect string value: '\\xEE\\xBD\\xD6\\x96\\x00\\x02...' for column `contao413`.`tl_lead_data`.`value` at row 1 at .../vendor/doctrine/dbal/src/Driver/PDO/Statement.php:121)"} []
```

**Comment by Toflar** [comment]
New features: FineUploader, the script, has reached EOL already 5 years ago (https://github.com/FineUploader/fine-uploader/issues/2073), so there's absolutely no point in adding any new features. At some point, we'll have to archive this extension completely.

Support: Yes, for now we still fix bugs that we encounter in our own customer projects. So it still is a supported extension but apparently none of us encountered your issue yet. Feel free to hire someone to look into it, if you need a solution now. We're also available for hire - see https://www.terminal42.ch/en/open-source for more info.

**Comment by LBeckX** [comment]
I saw that a pull request from Dez. 2022 was not included. If I make a pull request, will it be accepted if it is OK?
https://github.com/terminal42/contao-fineuploader/pull/94

**Comment by Toflar** [comment]
Yeah sure, if the quality is good and we find time to merge it, we'll do so. And in the meantime, use the power of it being open source - just stick to your own fork and switch back to the main repository as soon as it's ready 😊

**Comment by fritzmg** [comment]
There have been some Contao 5 fixes now in `3.4.9`.
